doc/nrf/security/crypto/crypto_supported_features.rst

```diff
@@ -568,9 +568,128 @@ The options are grouped by Series and drivers available for the device Series, a
                  - :kconfig:option:`CONFIG_PSA_WANT_KEY_TYPE_ML_KEM_PUBLIC_KEY`
                  - Supported
 
-   .. tab:: nRF54 Series
+   .. tab:: nRF54H Series
 
-      The following tables list the supported key types for nRF54 Series devices.
+      The following tables list the supported key types for nRF54H Series devices when running the :ref:`IronSide Secure Element <ug_nrf54h20_ironside>` on the Secure Domain.
```

Comment:
```suggestion
      The following tables list the supported key types for nRF54H Series devices.
```

If you agree, then we could simplify this sentence by dropping the "when", since one must always run the ISE on the secure domain.